src/components/network/index.js

```diff
@@ -99,24 +100,39 @@ class Network {
     this.bitswap._onPeerDisconnected(peerInfo.id)
   }
 
-  // Connect to the given peer
   connectTo (peerId, callback) {
-    const done = (err) => setImmediate(() => callback(err))
-
     if (!this._running) {
-      return done(new Error('No running network'))
+      return callback(new Error('No running network'))
     }
 
-    // NOTE: For now, all this does is ensure that we are
-    // connected. Once we have Peer Routing, we will be able
-    // to find the Peer
-    if (this.libp2p.swarm.muxedConns[peerId.toB58String()]) {
-      done()
-    } else {
-      done(new Error('Could not connect to peer with peerId:', peerId.toB58String()))
-    }
+    this.libp2p.dial(peerId, (err, conn) => {
+      if (err) {
+        return callback(err)
+      }
+
+      pull(pull.empty, conn)
+      callback()
+    })
+  }
```

Comment:
Dialing without a protocol doesn't create a `conn` because it never opens a multiplex conn, it is what is called 'warming up a conn'.

This can be just: `this.libp2p.dial(peerId, callback)`